src/sentry/api/serializers/models/release.py

```diff
@@ -369,6 +369,7 @@ def expose_project(project):
                 "id": project["id"],
                 "slug": project["slug"],
                 "name": project["name"],
+                "newGroups": project["new_groups"],
```

Comment:
Should we use `issues` which is the customer facing name?

Reply:
I would like to remain consistent because API returns it as newGroups everywhere else in the releases. In the UI we label it as New Issues.